libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -159,8 +160,36 @@ func UseSystemd() bool {
 			}
 		}
 
+		// Assume we have the ability to start a transient unit as a slice
+		// This was broken until systemd v229, but has been back-ported on RHEL environments >= 219
+		// For details, see: https://bugzilla.redhat.com/show_bug.cgi?id=1370299
```

Comment:
@cyphar - by broken, I mean you would literally get an error back from systemd when you called start transient unit where the unit was a `.slice`.  

The error response had the text: `Unit type slice does not support transient units`. 

This was fixed in https://github.com/systemd/systemd/commit/17f62e9bd00f5fefd486475861b06d3ec6b7ee10

As part of https://github.com/systemd/systemd/pull/1886

The code addresses this by:

1. attempting to start a slice, and if successful, it sets hasStartTransientUnitSlice to true.
1. if it detects you cannot, and you later attempt to start a transient unit of type slice, it will error early with the message that your systemd level did not support the feature yet.

Does that help?